src/main/java/com/paydesk/controller/HelloWorldController.java

```diff
@@ -0,0 +1,22 @@
+package com.paydesk.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.ModelAndView;
+ 
+@Controller
+public class HelloWorldController {

```

Comment:
We can have it . But its just an example class . We can have test class for actual controller classes